flink-fs-tests/src/test/java/org/apache/flink/hdfstests/HDFSTest.java

```diff
@@ -257,7 +265,88 @@ public void testBlobServerRecovery() throws Exception {
 		try {
 			blobStoreService = BlobUtils.createBlobStoreFromConfig(config);
 
-			BlobRecoveryITCase.testBlobServerRecovery(config, blobStoreService);
+			BlobServerRecoveryTest.testBlobServerRecovery(config, blobStoreService);
+		} finally {
+			if (blobStoreService != null) {
+				blobStoreService.closeAndCleanupAllData();
+			}
+		}
+	}
+
+	/**
+	 * Tests that with {@link HighAvailabilityMode#ZOOKEEPER} distributed corrupted JARs are
+	 * recognised during the download via a {@link org.apache.flink.runtime.blob.BlobServer}.
+	 */
+	@Test
+	public void testBlobServerCorruptedFile() throws Exception {
+		org.apache.flink.configuration.Configuration
+			config = new org.apache.flink.configuration.Configuration();
+		config.setString(HighAvailabilityOptions.HA_MODE, "ZOOKEEPER");
+		config.setString(CoreOptions.STATE_BACKEND, "ZOOKEEPER");
```

Comment:
I'm pretty sure, this should have been `FILESYSTEM` and I was just to eager to change everything to `ZOOKEEPER` when copying the tests over... Anyway, this option is not used at all because we only test the BLOB service classes (also in the test classes I copied this from).